Expanding Space/Assets/Scripts/Camera/CameraMovement.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraMovement : MonoBehaviour
+{
+    
+    public float movementSpeed = 0;
+    public float topMovementSpeed = 0;
+    void FixedUpdate()
+    {
+
+        transform.position += new Vector3(movementSpeed, 0, 0) * Time.deltaTime;
+
+        // this if statement makes the camera slowely go to the top speed 
+        if (movementSpeed <= topMovementSpeed)
+        {
+            movementSpeed += 0.01f;
```

Comment:
Misschien van `0.01f` een public float maken zodat we het in de inspector kunnen aanpassen als dat nodig is